schematic/utils/schema_utils.py

```diff
@@ -2,13 +2,20 @@
 import json
 import logging
 import networkx as nx
+import re
 import string
-from typing import Dict, List, Literal
+from typing import List, Literal, Dict, Tuple, Union
+
 
 logger = logging.getLogger(__name__)
 
 DisplayLabelType = Literal["class_label", "display_label"]
 BLACKLISTED_CHARS = ["(", ")", ".", " ", "-"]
+DELIMITERS = {
+    "component_name_delimiter": "#",
+    "component_rules_delimiter": "^^",
+    "rule_delimiter": "::",
+}
```

Comment:
As mentioned in code review, I think the delimeters would be better as their own globals since you don't use the dict for anything besides looking up a specific value.